README.md

```diff
@@ -1 +1 @@
-# self-review
```

Comment:
### Making a professional readme 
The most important rule in maintaining a professional repo is to always think about people that you share your code with: collaborators, code reviewers, programmers, recruiters or non-technical users who might want to reuse and/or improve your project.

**I would advice you to make your readme professional for reviewer and collaborator to be easy undertand what you are doing** 

- Resource to make your readme professional [Readme rule](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- If you want to make your GitHub messages more organized you can use [Markdown syntax](https://guides.github.com/features/mastering-markdown/).